File: server/atm-finder-service/src/main/java/otp/junctionx/atm/finder/dto/AtmResponse.java
package otp.junctionx.atm.finder.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.NoArgsConstructor;
import otp.junctionx.atm.finder.dto.helper.Address;
import otp.junctionx.atm.finder.dto.helper.Coord;

@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AtmResponse {

    public String id;
    public String date;
    public String day;
    public Boolean isDepositAvailable;
    public Address address;
    public Coord coord;
    public Integer countOfFutureCustomers;
    public String queueAndTravelTime;

}

File: client/app/src/main/java/hu/bme/aut/otpsmartatm/network/model/AtmResponse.kt
package hu.bme.aut.otpsmartatm.network.model

import com.google.gson.annotations.SerializedName

data class AtmResponse(
    val address: Address,
    @SerializedName("coord")
    val coordinate: Coordinate,
    val countOfFutureCustomers: Int,
    val date: String,
    val day: String,
    val id: String,
    val isDepositAvailable: Boolean,
    val queueAndTravelTime: String
)